hack/local-up-cluster.sh

```diff
@@ -64,7 +64,13 @@ do
 done
 
 if [ "x$GO_OUT" == "x" ]; then
-    "${KUBE_ROOT}/hack/build-go.sh" cmd/kube-proxy cmd/kube-apiserver cmd/kube-controller-manager cmd/kubelet plugin/cmd/kube-scheduler
+    "${KUBE_ROOT}/hack/build-go.sh" \
+        cmd/kube-proxy \
+        cmd/kube-apiserver \
+        cmd/kube-controller-manager \
+        cmd/kubelet \
+        plugin/cmd/kube-scheduler \
+        cmd/kubectl
```

Comment:
sorted and pushed

On Tue, Dec 15, 2015 at 3:05 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In hack/local-up-cluster.sh
> <https://github.com/kubernetes/kubernetes/pull/18692#discussion_r47622939>
> :
>
> > @@ -64,7 +64,13 @@ do
> >  done
> >
> >  if [ "x$GO_OUT" == "x" ]; then
> > -    "${KUBE_ROOT}/hack/build-go.sh" cmd/kube-proxy cmd/kube-apiserver cmd/kube-controller-manager cmd/kubelet plugin/cmd/kube-scheduler
> > +    "${KUBE_ROOT}/hack/build-go.sh" \
> > +        cmd/kube-proxy \
> > +        cmd/kube-apiserver \
> > +        cmd/kube-controller-manager \
> > +        cmd/kubelet \
> > +        plugin/cmd/kube-scheduler \
> > +        cmd/kubectl
>
> nit: can you please sort them alphabetically?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18692/files#r47622939>.
>